js/contraster.js

```diff
@@ -16,6 +16,11 @@ bcpl.contraster = (($, browserStorage) => {
 	const contrasterSettings = {};
 
 	const localStorageHighContrastKey = 'isHighContrast';
+	const isHighContrast = localStorage.isHighContrast === 'true';
```

Comment:
if we have the key above, you should probably make this line:

```javascript
const isHighContrast = localStorage[localStorageHighContrastKey ] === 'true';
```

Reply:
Okay, I'll make that change

Reply:
I think maybe you should be using `localStorage.getItem(key) ?`

Reply:
Yeah, you are right.

Reply:
Done